apps/settings/locales/settings.en-US.properties

```diff
@@ -1507,17 +1508,17 @@ settings-alert-dialog-header={{settings-dialog-alert}}
 settings-confirm-dialog-header={{settings-dialog-confirm}}
 settings-prompt-dialog-header={{settings-dialog-prompt}}
 
-accessible-app-icon.ariaLabel=Icon for {{ appName }} app
+accessible-app-icon.ariaLabel=Icon for {{appName}} app
```

Comment:
to avoid any string change side-effect , file bug 1220532 (as good first bug) to handle this nit